roles/ceph-grafana/tasks/setup_container.yml

```diff
@@ -9,6 +9,24 @@
   with_items:
     - /etc/grafana
     - /var/lib/grafana
+    - /var/lib/grafana/plugins
+
+- name: Install grafana plugins
+  shell: |
+    wget -nv "https://grafana.com/api/plugins/{{ item }}/versions/latest/download" -O "/tmp/{{ item }}.zip"
```

Comment:
you might want to use unarchive module ?
https://docs.ansible.com/ansible/latest/modules/unarchive_module.html